includes/solr_results.inc

```diff
@@ -382,7 +382,10 @@ class IslandoraSolrResultsBookmark extends IslandoraSolrResults {
     if (array_key_exists($key, $form_state['values'])) {
       $params[] = $form_state['values'][$key];
     }
```

Comment:
I missed this before but we need to have an else here that adds a NULL to keep the positioning of the next param consistent.